src/bioregistry/data/bioregistry.json

```diff
@@ -14623,6 +14623,38 @@
       "uri_format": "https://www.cellrepo.com/repositories/$1"
     }
   },
+  "cellxgene": {
+    "contributor": {
+      "email": "robert.j.allaway@gmail.com",
+      "github": "allaway",
+      "name": "Robert Allaway",
+      "orcid": "0000-0003-3573-3565"
+    },
+    "description": "CELLxGENE is an interactive data visualization and exploration tool developed by the Chan Zuckerberg Initiative that enables researchers to analyze and share single-cell genomics datasets. It provides a user-friendly interface for biologists and computational scientists to interrogate gene expression patterns across different cell types. Identifiers correspond to collections made available in CELLxGENE.",
+    "example": "436154da-bcf1-4130-9c8b-120ff9a888f2",
+    "github_request_issue": 1541,
+    "homepage": "https://cellxgene.cziscience.com",
+    "license": "CC-BY-4.0",
+    "name": "CZ CELLxGENE",
+    "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$",
+    "publications": [
+      {
+        "doi": "10.1093/nar/gkae1142",
+        "pmc": "PMC11701654",
+        "pubmed": "39607691",
+        "title": "CZ CELLxGENE Discover: a single-cell data platform for scalable exploration, analysis and modeling of aggregated data",
+        "year": 2024
+      }
+    ],
+    "repository": "https://github.com/chanzuckerberg/single-cell-data-portal",
```

Comment:
please remove this - it's a bit misleading since the semantic space isn't curated within this repository